data/mods/Aftershock/items/books.json

```
@@ -17,6 +17,38 @@
     "chapters": 30,
     "fun": 6
   },
+  {
+    "id": "afs_welcome1_book",
+    "type": "BOOK",
+    "name": { "str": "Unofficial History of Salus IV Vol. 1", "str_pl": "copies of Unofficial History of Salus IV Vol. 1" },
+    "description": "Welcome to Frozen Hell, an Unofficial History of Salus IV by Bette Lapin.  Salus IV was founded in the early days of the Hyperspace Era by the Interstellar Soviet Community.  At it's height several hundred years ago Salus IV had just over one billion sentient citizens, but that population had begun to decline even before the Discontinuity as people migrated to planets that while more hostile to life during the earliest colonization phases than Salus IV were much simpler to terraform into a place more enjoyable by human and uplift citizens.",
```

```suggestion
    "description": "Welcome to Frozen Hell, an unofficial history of Salus IV by Bette Lapin.  Salus IV was founded in the early days of the Hyperspace Era by the interstellar Soviet community.  At its height several hundred years ago Salus IV had just over one billion sentient citizens, but that population had begun to decline even before the Discontinuity as people migrated to planets that while more hostile to life during the earliest colonization phases than Salus IV were much simpler to terraform into a place more enjoyable by human and uplift citizens.",
```

Just a minor tidbit about this about the lore implications, why one billion citizens? sounds like a big number, especially for a world that is on the edge like Salus IV

As well the tidbit about Soviet community, no objections if we want this historically to be a soviet colony, but I believed the idea was the soviet involvement was just help setting up infrastructure and everything, right?

---

I thought it was made on behalf of  Whately Corporation yeah, in fact the timeline still says this, although that was only me deciding this (maybe based on idle discord conversation)

> Near the middle of the colonization craze in 2102, the Whately Corporation claims the Salus System and begins building a science outpost on the viable planets.

---

I'm ok either way.  I saw some things I thought were contradicting each other in the timeline and tried to adjust to make it all make sense but I think that was my misunderstanding that soviet infrastructure was used rather than it was an explicitly soviet colony.